Tests/Unit/Domain/Search/QueryFactoryTest.php

```diff
@@ -543,7 +542,7 @@ public function scriptFieldsAreAddedToQuery()
                         'config' => 'something',
                     ],
                     'field2' => [
-                        'config' => '{request.query}',
+                        'config' => '{request.searchTerm}',
```

Comment:
Is this change still valid? Check if you want to use `{request.query}` or `{request.searchTerm}`.